pkg/client/image/image.go

```diff
@@ -35,6 +41,40 @@ func (c *Client) Delete(id string) error {
 	return c.client.Delete(id, c.instanceID)
 }
 
+//func ImportImage imports image from S3 Instance
+func (c *Client) ImportImage(instanceID, imageName, s3Filename, region, accessKey, secretKey, bucketName, osType, diskType string) (*models.Image, error) {
```

Comment:
shouldn't diskType be storageType to be in sync ?

Reply:
@bkhadars ^^

Reply:
changed to storageType